wagtail/tests/test_org_block.py

```diff
@@ -0,0 +1,31 @@
+import os
```

Comment:
Please use the same structure for tests as used elsewhere in the project - a `TestCase` subclass containing assertions. We want to be able to run these as an automated test suite, and not have to run scripts individually.